rllib/examples/parallel_evaluation_and_training.py

```diff
@@ -134,9 +134,6 @@ def on_train_result(self, *, trainer, result, **kwargs):
         # Evaluate every other training iteration (together
         # with every other call to Trainer.train()).
         "evaluation_interval": args.evaluation_interval,
-        "evaluation_config": {
```

Comment:
Commenting for future reference, but that line of code wasn't actually doing anything, since eval workers don't read from offline data. Will be fixed in another PR soon.

Reply:
Yes, makes sense. Sampler -> No evaluation. Cool, I think that also answers the question on the default value of `off_policy_estimation_methods` being empty list (I think we can leave this as by default (input=sampler), no results are generate anyways).